extensions/wikia/DesignSystem/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "app-design-system",
   "scripts": {
-    "setup": "npm install && node i18n/build && SERVER_ID=177 php ../../../maintenance/rebuildLocalisationCache.php --force --thread=2 --primary",
```

Comment:
Do we not need to run `rebuildLocalisationCache` during set-up anymore?

Reply:
on new devboxes i does not work, you need this long docker command to do this, If you want I can paste it here

Reply:
Ok, good to know!